db/schema.rb

```diff
@@ -845,6 +867,21 @@
     t.integer  "jnlp_session_id"
   end
 
+  create_table "interactives", :force => true do |t|
+    t.string   "name"
+    t.text     "description"
+    t.string   "url"
+    t.integer  "width"
+    t.integer  "height"
+    t.float    "scale"
+    t.string   "image_url"
+    t.integer  "user_id"
+    t.string   "credits"
+    t.string   "publication_status"
+    t.datetime "created_at",         :null => false
+    t.datetime "updated_at",         :null => false
+  end
+
```

Comment:
I think this create_table statement got included accidentally -- I don't see a migration for it.